cmake-tests/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ endif()
 set_target_properties(cmake-test PROPERTIES LINK_FLAGS ${new_cmake_test_link_flags})
 
 if(TARGET hccrt)
+    add_dependencies(cmake-test clang_links rocdl_links)
```

Comment:
No, this add_dependencies is about 'cmake-test' target that is described in the file. It was moved into 'if(TARGET hccrt)' since the dependencies should not affect standalone build of 'cmake-test'